src/Groups/libraries/atlasgroups.jl

```diff
@@ -44,6 +49,164 @@ function atlas_group(::Type{T}, name::String) where T <: Union{PermGroup, Matrix
   return TT(G)
 end
 
+
+"""
+    atlas_group(info::Dict)
+
+Return the group from the Atlas of Group Representations
+that is defined by `info`.
+Typically, `info` is returned by [`all_atlas_group_infos`](@ref).
```

Comment:
Maybe more like this?
```suggestion
Typically, `info` is obtained from [`all_atlas_group_infos`](@ref).
```